ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -137,6 +138,13 @@ get_data_time_str(const double data_time, const double current_time, const doubl
     }
 }
 
+/*!
+ * Get the smallest cell width on the specified level. This operation is
+ * collective.
+ */
+double get_min_patch_dx(const SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM> > hierarchy,
```

Comment:
Not crucial, but it would be a bit more general to take in a `PatchLevel` (since one can have a `PatchLevel` that is not in a `PatchHierarchy`).